### PR TITLE
grub-bios-setup: add page

### PR DESCRIPTION
**Pull request by nebulaa44**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Resolves part of issue #8080

### CODE REVIEW

File: pages/linux/grub-bios-setup.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# grub-bios-setup
+
+> Set up a BIOS device to use GRUB.
```
Comment (kbdharun) [inline pages/linux/grub-bios-setup.md:3]:
```suggestion
> Set up a device to use GRUB.
```

**Review by kbdharun — CHANGES_REQUESTED** [review]

**Comment by kbdharun** [comment]
@CleanMachine1 Should we add these flags too to the page?

```
grub-bios-setup --boot-image=FILE
Use FILE as the boot image. The default value is boot.img.

grub-bios-setup --core-image=FILE
Use FILE as ther core image. The default value is core.img.

grub-bios-setup --directory=DIR
Use GRUB files in the directory DIR. The default value is /boot/grub.

grub-bios-setup --force
Install even if problems are detected.

grub-bios-setup --skip-fs-probe
Do not probe DEVICE for filesystems.

```
Ref: [Manned](https://manned.org/grub-bios-setup.8) , [Ubuntu docs](https://manpages.ubuntu.com/manpages/jammy/man8/grub-bios-setup.8.html)

File: pages/linux/grub-bios-setup.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# grub-bios-setup
+
+> Set up a device to use GRUB.
+> You should use `grub-install` instead of `grub-bios-setup` in most cases.
+> More information: <https://manned.org/grub-bios-setup.8>.
+
+- Set up a device to boot with GRUB:
+
+`grub-bios-setup {{/path/to/device/file}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/grub-bios-setup.md:9]:
```suggestion
`grub-bios-setup {{/dev/sdX}}`
```

```suggestion
`grub-bios-setup {{/path/to/device}}`
```

Choose one of two suggestions.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/linux/grub-bios-setup.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# grub-bios-setup
+
+> Set up a device to use GRUB.
```
Comment (CleanMachine1) [inline pages/linux/grub-bios-setup.md:3]:
```suggestion
> Set up a device to use GRUB with a BIOS configuration.
```

**Review by kbdharun — COMMENTED** [review]
I am fine with these changes. I just want to know whether other flags are required or if this command is alone enough.
What's your opinion @navarroaxel @CleanMachine1?

**Comment by navarroaxel** [comment]
> I am fine with these changes. I just want to know whether other flags are required or if this command is alone enough. What's your opinion @navarroaxel @CleanMachine1?

The other flags are a nice to have, but no a `must` to merge. Let's wait an answer from @nebulaa44.

**Comment by nebulaa44** [comment]
> > I am fine with these changes. I just want to know whether other flags are required or if this command is alone enough. What's your opinion @navarroaxel @CleanMachine1?
> 
> 
> 
> The other flags are a nice to have, but no a `must` to merge. Let's wait an answer from @nebulaa44.

I don't mind adding more examples. I'm away from my PC at the moment, so it'll be around an hour or so.

**Comment by nebulaa44** [comment]
I thought I had committed the second commit as `add --force example` but apparently git wanted to ignore that. Sorry about that :P

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Pull request merged**